src/intertypes/json.jl

```diff
@@ -0,0 +1,237 @@
+export intertype_to_jsonschema, jsonwrite, jsonread, parse_intertype, toexpr, @intertype_decls, as_intertypes
+
+using OrderedCollections
+using Base64
+using MLStyle
+import JSON3
+
+struct JSONFormat
```

Comment:
```julia
abstract type SerialFormat end
Struct JSONFormat <: SerialFormat end
```
?
Suppose it doesn’t matter if no other Serial formats are on the radar.

Reply:
Eventually, I also want s-expression and msgpack serialization